api/src/modules/scenarios/dto/planning-units-update.dto.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  IsOptional,
+  IsUUID,
+  ValidateIf,
+  ValidateNested,
+} from 'class-validator';
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { FeatureCollection } from 'geojson';
+import { Type } from 'class-transformer';
+import { IsFeatureCollectionOfPolygons } from './is-feature-collection-of-polygons';
+
+export class PlanningUnitsByIdUpdateDto {
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsUUID('all', { each: true })
+  include?: string[];
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsUUID('all', { each: true })
+  exclude?: string[];
+}
+
+export class PlanningUnitsByGeoJsonUpdateDto {
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsFeatureCollectionOfPolygons({ each: true })
+  include?: FeatureCollection[];
+
+  @ApiPropertyOptional()
+  @IsOptional()
+  @IsFeatureCollectionOfPolygons({ each: true })
+  exclude?: FeatureCollection[];
+}
+
+export class PlanningUnitsUpdateDto {
```

Comment:
@hotzevzl we are neither updating their status(es) 😅 the DTO does not always need to match the entity, especially in case there isn't one.

How about `ChangeScenarioPlanningUnitsInclusivityDto`?

If not convinced, I can change it to your suggestion.

Reply:
what about `UpdateScenarioPlanningUnitLockStatusDTO` then? 😅 

I'm happy with the rest of the PR after your change to allow to handle both ids and geojsons, so I'll leave the final choice on this to you - after all we all use words in slightly different ways, so there's no perfectly denotative names for everyone, and I acknowledge I am particularly fastidious with naming things 😅 

my loose argument - we are asking to update something (not necessarily changing), and what we are updating is called "status" in Marxan. I think any of the names in this thread more closely match the intention than the original name... roll your dice!

agree that DTOs don't necessarily match entities - that's actually the point of DTOs in general! I meant entity in a general sense (guilty of misusing words here!) of something we act on (resource in REST terms?) which could be relying on multiple "db-entities" behind the scenes - but I think we're more or less aligned on this - my comment was mainly about the order of action/thing acted upon/DTO, let's say 😄

Reply:
Changed in https://github.com/Vizzuality/marxan-cloud/pull/151/commits/7c5f7d470aa6a3616bdb07f8690b1766ff8436cf